pkg/config/workloads/workloads.go

```diff
@@ -0,0 +1,136 @@
+package workloads
```

Comment:
please don't judge me too hard for the current impl, I am not yet proud of it :)